project.clj

```diff
@@ -45,15 +45,20 @@
                          [org.apache.commons/commons-lang3 "3.4"]
                          [org.apache.httpcomponents/httpclient  "4.5.2"]
                          [org.apache.httpcomponents/httpcore  "4.4.5"]
+                         [commons-beanutils "1.9.2"]
                          [commons-codec "1.10"]
+                         [commons-collections "3.2.2"]
                          [commons-lang "2.6"]
+                         [commons-logging "1.2"]
                          [commons-io "2.4"]
                          [joda-time "2.8.2"]
 
+                         [bidi "2.0.12"]
```

Comment:
This will allow us to remove the bidi binding in https://github.com/puppetlabs/comidi/blob/master/project.clj#L14 as well (assuming we update the comidi dep on clj-parent).